app/views/search/_filters.html.slim

```diff
@@ -0,0 +1,64 @@
+.col-xs-12.col-sm-3
+  .filters
+    .filters__title.text-xs-left.text-center
+      h6
+        | Filter by category
+        span.filters__clear= link_to 'Clear', root_path, class: 'gray-light pull-right'
+    .filters__lists
+      = hidden_field_tag :query, @query
+      ul.filters__list
+        li 
+          .filters__item
+            = render 'search/filter', name: :resources, category: :model_types, indeterminate: true
+          ul.filters__list
+            li.filters__item
+              = render 'search/filter', name: :books, category: :resource_types
+            li.filters__item
+              = render 'search/filter', name: :articles, category: :resource_types
+            li.filters__item
+              = render 'search/filter', name: :reports, category: :resource_types
+        li.filters__item
+          = render 'search/filter', name: :lists, category: :model_types
+        li.filters__item
+          = render 'search/filter', name: :groups, category: :model_types
+      .clearfix
+      
+  .filters
+    .filters__title.text-xs-left.text-center
+      h6
+        | Accessibility
+        span.filters__clear= link_to 'Clear', root_path, class: 'gray-light pull-right'
+    .filters__lists
+      = hidden_field_tag :query, @query
+      ul.filters__list
+        li 
+          .filters__item
+            = render 'search/filter', name: :free, category: :model_types, indeterminate: true
+        li.filters__item
+          = render 'search/filter', name: :Downloadable, category: :model_types
+      .clearfix
+      
+  .filters
+    .filters__title.text-xs-left.text-center
+      h6
+        | Tags
+        span.filters__clear= link_to 'Clear', root_path, class: 'gray-light pull-right'
+    .tags-list
```

Comment:
I'm already getting the list of all the tags present in the search results (to provide the suggestions). We will need to change/add a bit of logic to get the top tags and count the occurrences. But I'm wondering what happens when a user clicks on a tag?